content/docs/authorization/troubleshooting.md

```diff
@@ -52,20 +52,6 @@ For OPA related logs, run:
 $ k3s kubectl logs deploy/proxy-server -n karavi -c opa
 ```
 
-### Running "karavictl tenant" commands result in an HTTP 504 error
```

Comment:
Added it back. The error message may not look like what is documented because it would be an http error rather than rpc, but leaving it like this for now.